docs/src/ref_impl/labwc_config.md

```diff
@@ -0,0 +1,23 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
```

Comment:
Please add here a subsection "#Overview" or "#Rationale" that summarizes why this concept has been developed.

A few keys to help you:
- border color concept illustrates the app trustworthiness
- please attribute earlier work of the concept to https://www.qubes-os.org/doc/getting-started/#color--security (possible other before QubesOS)
- list the limitations of current implementation - e.g. if the trustworthiness color is forgeable and other things you see that need further development

Reply:
This part of the documentation would also benefit of screencapture image.